src/client/src/containers/DependencyInfo/index.tsx

```diff
@@ -0,0 +1,128 @@
+import * as React from "react";
+import { connect } from "react-redux";
+import styles from "./styles.module.css";
+import * as getSvg from "../../utils/getSvgUrl";
+import classnames from "classnames";
+import { injectIntl, defineMessages } from "react-intl";
+import { WIZARD_CONTENT_INTERNAL_NAMES } from "../../utils/constants";
+import { AppState } from "../../reducers";
+import { IDependenciesInstalled } from "../../reducers/dependencyInfoReducers";
+
+interface IDependencyInfoProps {
+  frameworkName: string;
+  intl: any;
+  dependenciesStore: IDependenciesInstalled;
+}
+
+type Props = IDependencyInfoProps;
+
+const messages = defineMessages({
+  installed: {
+    id: "dependencyChecker.installed",
+    defaultMessage: " detected!"
+  },
+  notInstalled: {
+    id: "dependencyChecker.notInstalled",
+    defaultMessage: " not detected. Click to install."
+  },
+  iconAltMessage: {
+    id: "dependencyChecker.iconAltMessage",
+    defaultMessage: "Icon for dependency checker"
+  }
+});
+
+interface IDependency {
+  [key: string]: string;
```

Comment:
Feel like you should make this a bit more solid. Like dependencyName, displayName, downloadLink, privacyStatementLink.